qutip/visualization.py

```diff
@@ -171,33 +183,27 @@ def _cb_labels(left_dims):
 
 
 # Adopted from the SciPy Cookbook.
-def hinton(rho, xlabels=None, ylabels=None, title=None, ax=None, cmap=None,
-           label_top=True, color_style="scaled"):
+def hinton(rho, xticklabels=None, yticklabels=None, cmap = None, colorbar = True,
+           color_style="scaled", label_top=True, figure = None, axes = None):
```

Comment:
Have you though about using keyword only arguments?
`hinton(rho, ..., *, figure=None, axes=None)`
Maybe setting `figure` etc. as keyword only would make sense.